crowbar_framework/app/models/service_object.rb

```diff
@@ -282,7 +282,7 @@ def destroy_active(inst)
       dep = role.override_attributes
       dep[@bc_name]["elements"] = {}
       dep[@bc_name].delete("elements_expanded")
-      @logger.debug "#{inst} proposal has a crowbar-committing key" if dep[@bc_name]["config"].has_key? "crowbar-committing"
+      @logger.debug "#{inst} proposal has a crowbar-committing key" if dep[@bc_name]["config"].key? "crowbar-committing"
```

Comment:
Metrics/LineLength: Line is too long. [120/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)